infrastructure/tooling/src/build/tasks/livelog.js

```diff
@@ -72,9 +72,16 @@ module.exports = ({ tasks, cmdOptions, credentials, baseDir, logsDir }) => {
       // this simple Dockerfile just packages the binary into a Docker image
       const dockerfile = path.join(contextDir, 'Dockerfile');
       fs.writeFileSync(dockerfile, [
-        'FROM busybox', // We use busybox rather than scratch to have `/tmp` and such things
+        'FROM ubuntu:latest AS certs',
+        'RUN apt-get update',
+        'RUN apt-get upgrade -y',
+        'RUN apt-get install -y ca-certificates',
+        'RUN mkdir /empty',
```

Comment:
These _could be_ reduced to one `RUN` command to reduce the layers, but not too big of a deal here I don't think. Up to you!

Reply:
we don't keep the image, so i don't think it makes a difference :-)

we throw it away, and create a new image from scratch :-)